wordpress-mobile/gutenberg-mobile
# Fix: Media & Text Loses upload status if post is closed/reopened during the upload #18137

Fixes 
wordpress-mobile/gutenberg-mobile#1504
 https://github.com/wordpress-mobile/gutenberg-mobile/issues/1508

To test:
See gutenberg PR https://github.com/WordPress/gutenberg/pull/18137

Update release notes:
